trading212/models.py

```diff
@@ -183,32 +185,32 @@ class Tax(BaseModel):
 
 class HistoryItem(BaseModel):
     dateCreated: datetime
-    dateExecuted: datetime
+    dateExecuted: datetime | None
     dateModified: datetime
     executor: str
-    fillCost: float
-    fillId: int
-    fillPrice: float
-    fillResult: float
-    fillType: str
-    filledQuantity: float
-    filledValue: float
+    fillCost: float | None
+    fillId: int | None
+    fillPrice: float | None
+    fillResult: float | None
+    fillType: str | None
+    filledQuantity: float | None
+    filledValue: float | None
```

Comment:
@jogobeny Thanks for the contribution. Would you mind using the `Optional` field type instead?

```suggestion
    fillCost: Optional[float] = None
    fillId: Optional[int] = None
    fillPrice: Optional[float] = None
    fillResult: Optional[float] = None
    fillType: Optional[str] = None
    filledQuantity: Optional[float] = None
    filledValue: Optional[float] = None
```